extensions/static-marks-bookmarks/package.json

```diff
@@ -22,27 +25,27 @@
   "preferences": [
     {
       "name": "static-mark-yaml-url",
-      "description": "The url where your Static Marks YAML file is located",
+      "description": "The absolute path or public url where your Static Marks YAML file is located",
       "type": "textfield",
-      "placeholder": "https://raw.githubusercontent.com/../bookmarks.yml",
-      "required": true,
-      "title": "Static Marks YAML URL"
+      "placeholder": "/Users/[user folder]/bookmarks.yml",
+      "required": false,
```

Comment:
Why mark this as not required? Isn't it needed for the extension to work?

Reply:
Was gonna say this. Seems like it should be required: 
<img width="873" alt="image" src="https://user-images.githubusercontent.com/372831/201920468-8c42af92-642d-4bcf-aa91-2b0ee58a3772.png">

Reply:
You're right, I played with the idea of making it default to the url placeholder, but finally went with the more explicit behaviour. Reverting that change right away.

Reply:
Thanks for improving the extension btw. 😄 looks really cool!